docs/0-installation.md

```diff
@@ -21,6 +26,11 @@ rails g active_admin:install              # creates the AdminUser class
 rails g active_admin:install User         # creates / edits the class for use with Devise
 rails g active_admin:install --skip-users # skips Devise install
 ```
+If you choose to use Devise, just add it to your Gemfile :
+```ruby
+gem 'devise'
+```
+and run `bundle install`
```

Comment:
That's a great idea, but I never used Active Admin without Devise...

Can I let the "Without Devise" paragraph in progress ?